--- origsrc/pycairo-1.8.6/configure.ac	2009-06-25 03:38:07.000000000 -0500
+++ src/pycairo-1.8.6/configure.ac	2009-08-24 00:06:11.062255600 -0500
@@ -33,6 +33,8 @@ ACLOCAL="$ACLOCAL $ACLOCAL_FLAGS"
 # Checks for programs ---
 AC_DISABLE_STATIC
 AC_PROG_LIBTOOL dnl required version (1.4) DON'T REMOVE - used by autogen.sh
+dnl when using libtool 2.x create libtool early, because it's used in configure
+m4_ifdef([LT_OUTPUT], [LT_OUTPUT])
 
 AM_PATH_PYTHON(2.6)
 
